impeller/compiler/switches.cc

```diff
@@ -53,16 +53,17 @@ void Switches::PrintHelp(std::ostream& stream) {
   }
   stream << "}" << std::endl;
   stream << "--sl=<sl_output_file>" << std::endl;
-  stream << "--spirv=<spirv_output_file>" << std::endl;
+  stream << "--spirv=<spirv_output_file> (ignored for --shader-bundle)"
+         << std::endl;
   stream << "[optional] --source-language=glsl|hlsl (default: glsl)"
          << std::endl;
   stream << "[optional] --entry-point=<entry_point_name> (default: main; "
             "ignored for glsl)"
          << std::endl;
   stream << "[optional] --iplr (causes --sl file to be emitted in iplr format)"
          << std::endl;
-  stream << "[optional] --iplr-bundle=<bundle_spec> (causes --sl file to be "
-            "emitted in the iplr bundle format)"
+  stream << "[optional] --shader-bundle=<bundle_spec> (causes --sl file to be "
```

Comment:
Any thoughts about reading the json spec in from a file instead of the flag itself being json-valued? I wonder which will be easier for tools that wrap/call `impellerc`.

Reply:
My hunch is that storing it in a file before invoking is more of a pain given the ephemeral nature. Although it would be slightly more convenient for the many of manual invocations I've been doing over the past couple of days. : )